src/components/referral/ReferralCreditTable.js

```diff
@@ -3,7 +3,10 @@ import Table from "components/Table";
 
 const columnConfig = {
   cryptoAmount: {
-    children: "Crypto Amount"
+    children: "Crypto Amount",
+    dataFormat: (cell) => {
+      return Number(cell).toPrecision(3);
```

Comment:
```suggestion
      return Number(cell).toPrecision(8);
```
Shouldn't this be 8 according to the PR description?